addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -2094,7 +2098,7 @@ StartMusic(client=0)
 	{
 		StopMusic(client);
 		playBGM[client]=true;
-		CreateTimer(0.1, Timer_PrepareBGM, GetClientUserId(client), TIMER_FLAG_NO_MAPCHANGE);
+		CreateTimer(0.2, Timer_PrepareBGM, GetClientUserId(client), TIMER_FLAG_NO_MAPCHANGE);
```

Comment:
Why was this changed?

Reply:
Avoid bgm not playing again, but probably this is not the cause